src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 23px 0;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    color: $color-dark;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__link:hover {
+    color: $color-blue;
+  }
+
+  &__link.is-active {
+    color: $color-blue;
+
+    &::after {
+      position: absolute;
+      content: "";
+      height: 4px;
+      width: 100%;
+      bottom: 0;
+      background-color: $color-blue;
+      border-radius: 8px;
+    }
```

Comment:
It would be better to put all this stuff inside the &__link selector

Reply:
ok

Reply:
ok